app/src/main/java/org/mozilla/fenix/compose/SectionHeader.kt

```diff
@@ -20,20 +28,53 @@ import org.mozilla.fenix.theme.FirefoxTheme
 @Composable
 fun SectionHeader(
     text: String,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    appStore: AppStore = components.appStore
```

Comment:
This needs a change to a `color` parameter. Currently, `SectionHeader` is used in `MessageCard` so the changes to impose observing the `appStore` directly in this composable components will affect the reusability of this component.

Reply:
I have extracted the text header from the section header, creating a reusable `header7` composable.
This also uses a more correct naming as dialog, banners, and message cards do not have sections or section headers.

Do you also think we could rename `SimpleHeader` to a name that links it to the Fenix typography? Thank you!

Reply:
I also wonder if, in future refactoring, we should merge the `SectionHeader` with the `HomeSectionHeader` and conditionally handle the visibility of the show-all text.